__tests__/app.test.js

```diff
@@ -114,3 +114,35 @@ describe("/api/articles", () => {
       });
   });
 });
+
+describe("getComments", () => {
+  test("200: responds with an array of comments for the given article_id", () => {
+    //const articleId = 1
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(Array.isArray(comments)).toBe(true);
+        console.log(testData.commentData, "<--testData comments array");
```

Comment:
**please** remove console logs before submitting code for review. The test should be passing and no longer have a need for these logs.